.editorconfig

```diff
@@ -20,52 +20,3 @@ dotnet_diagnostic.MSML_NoBestFriendInternal.severity = none
 
 # MSML_NoInstanceInitializers: No initializers on instance fields or properties
 dotnet_diagnostic.MSML_NoInstanceInitializers.severity = none
-
-[test/Microsoft.ML.CodeAnalyzer.Tests/**.cs]
-# BaseTestClass does not apply for analyzer testing.
-# MSML_ExtendBaseTestClass: Test classes should be derived from BaseTestClass
-dotnet_diagnostic.MSML_ExtendBaseTestClass.severity = none
```

Comment:
📝 This one shouldn't change

Reply:
Yes, you are right, thanks

---
In reply to: [381542429](https://github.com/dotnet/machinelearning/pull/4858#discussion_r381542429) [](ancestors = 381542429)